pkg/util/iptables/iptables.go

```diff
@@ -519,6 +528,45 @@ func getIPTablesVersionString(exec utilexec.Interface) (string, error) {
 	return match[1], nil
 }
 
+// Checks if iptables-restore has a "wait" flag
+// --wait support landed in v1.6.1+ right before --version support, so
+// any version of iptables-restore that supports --version will also
+// support --wait
+func getIPTablesRestoreWaitFlag(vstring string) []string {
+	if vstring == "" {
+		return nil
+	}
+
+	if _, err := utilversion.ParseGeneric(vstring); err != nil {
+		glog.Errorf("vstring (%s) is not a valid version string: %v", vstring, err)
+		return nil
+	}
+
+	return []string{"--wait", "10"}
```

Comment:
The amount of time we might have to wait for the lock depends on whether the *other* operation is "iptables" or "iptables-restore", not on whether *this* operation is. So it doesn't make sense for this value to be different from the iptablesWaitFlag value; if 2 seconds isn't enough here then it's not enough there either. (And it's not enough in docker...)